samri/pipelines/manipulations.py

```diff
@@ -10,7 +10,8 @@ def collapse_nifti(in_dir, out_dir,
 	in_dir = os.path.abspath(os.path.expanduser(in_dir))
 	out_dir = os.path.abspath(os.path.expanduser(out_dir))
 	in_files = []
-	for root, dirs, files in os.walk(in_dir):
+	for root, dirs, files in os.walk(in_dir, followlinks=True):
+		for file in files:
```

Comment:
I don't understand, where does this for loop take place? There should be an indent after it, and some code using the variable `file`.

Reply:
yes the `for file in files` loop must be a leftover, sorry. I meant to add the `followlinks=True` exclusively in order to be able to follow symbolic links.

```suggestion
```